docs/administration-guide/databases/bigquery.md

```diff
@@ -30,3 +116,18 @@ As of version 0.30.0, Metabase tells BigQuery to interpret SQL queries as [Stand
 SELECT *
 FROM [my_dataset.my_table]
 ```
+
+## Troubleshooting
+
+If you're having trouble with your BigQuery connection, check out [Metabase's discussion forum](https://discourse.metabase.com/search?q=bigquery) to see if someone has encountered and resolved a similar issue.
+
+## Further reading
+
+- [Managing databases](https://www.metabase.com/docs/latest/administration-guide/01-managing-databases.html).
```

Comment:
This link is a bit redundant with the current article. It covers a lot of the same ground.